test/test_view.c

```diff
@@ -29,6 +29,7 @@ DEALINGS IN THE SOFTWARE.  */
 #include <unistd.h>
 #include <stdlib.h>
 #include <string.h>
+#include <getopt.h>
```

Comment:
As I remember, an ancient Sanger machine did not have getopt.h that time. I could be wrong, though. That was like over 10 years ago when I was even not sitting in an office at Sanger. I would be cautious of having "getopt.h" included.